src/components/ObsDetails/ActivityItem.js

```diff
@@ -48,6 +52,18 @@ const ActivityItem = ( {
     isActiveUserTheCurrentUser( );
   }, [user] );
 
+  const handleSuccess = {
+    onSuccess: ( ) => {
+      queryClient.invalidateQueries( ["fetchRemoteObservation", item.uuid] );
+      refetchRemoteObservation( );
+    }
+  };
```

Comment:
I know I harp on naming a lot, but a) this object literal doesn't really handle success, it's more like a configuration object for `useMutation()`, and b) is there a reason to define a separate constant here? Why not define the object literal when you call `useAuthenticatedMutation`?